docs/concepts/automatic-data-locality.md

```diff
@@ -24,10 +24,10 @@ metadata:
 
 ## Failing to move
 
-Under some specific scenarios a volume instance may not be able to follow a pod, I.E. into a compute-only node, thus remains where it current is.
+Under some scenarios a primary deployment may not be able to follow the pod. For example, if the pod moves to a compute-only node. In such scenario the primary deployment will remain where it currently is.
 
 The change is triggered again when the pod moves again though! And if needed, it can be [moved manually](/docs/concepts/move).
 
 ## Safety first
 
-The process is safe to use and only ever removes data when a new instance of it has been synced within the new node.
+The process is safe to use and only ever removes data when a new deployment of it has been synced within the new node.
```

Comment:
... when a new deployment ~of it~ has been synced ...

Reply:
Removed "of it"